rules/linux/process_creation/proc_creation_lnx_browser_information_discovery.yaml

```diff
@@ -0,0 +1,37 @@
+title: Browser Information Discovery - Linux
```

Comment:
```suggestion
title: Firefox Browser Information Discovery - Linux
```
As the references and filename are for firefox

Reply:
Its better to make this rule generic by adding additional files from other browsers instead of it just being firefox focused.

@CheraghiMilad please add additional chromium browsers.